evap/staff/templates/staff_semester_view.html

```diff
@@ -573,6 +578,15 @@ <h3>
             document.getElementById("form_activation_status").requestSubmit();
         }
     </script>
+    {% trans 'Create exam evaluation' as title %}
+    {% blocktrans asvar question %}Create an exam evaluation based on this evaluation. This will copy all the participants and contributors from the original evaluation. It will set the weight of the original evaluation to 9 and its end date will be set to the day before the exam. {% endblocktrans %}
+    {% trans 'Create exam evaluation' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='createExamEvaluation' title=title question=question action_text=action_text btn_type='danger' %}
+    <script type="text/javascript">
+        function createExamEvaluationAction() {
+            pass
```

Comment:
Seems like changes from another PR are included here, but you marked this PR as review ready -- do you expect us to review already or can I mark as draft here?